### PR TITLE
Add meta data getter

### PR DESCRIPTION
**Pull request by kolanos**

### CODE REVIEW

**Comment by kolanos** [comment]
@pselle Don't have perms to request review.

@coreylight For review.

File: src/index.js [hunk 1]
```diff
@@ -46,6 +46,11 @@ class TracePlugin {
       'post:setup': this.postSetup.bind(this),
       'pre:report': this.preReport.bind(this)
     };
+    this.meta = {
+      name: 'trace',
```
Comment (coreylight) [inline src/index.js]: @kolanos @pselle what do you think about prefixing our own, `iopipe-trace` in this case?

File: src/index.js [hunk 2]
```diff
@@ -46,6 +46,11 @@ class TracePlugin {
...
+      version: '0.1.2',
```
Comment (coreylight) [inline src/index.js]: @kolanos would you mind making a unit test that compares this value to the package.json value so we always keep them in line?

File: src/index.js (hunk 1 above)
Comment (pselle) [reply src/index.js]: ++

**Comment by coreylight** [comment]
@kolanos I think we should add a test for the meta property, since it's not executed in the tests as it stands.

File: src/index.test.js [hunk 3]
```diff
@@ -18,6 +19,9 @@ test('Can instantiate the plugin with no options', () => {
   expect(_.isPlainObject(inst.config)).toBe(true);
   expect(inst.timeline instanceof Perf).toBe(true);
   expect(inst.config.autoMeasure).toBe(true);
+  expect(inst.meta.name).toBe(pkg.name);
```
Comment (coreylight) [inline src/index.test.js]: @kolanos would it be cool to change pkg.name to a string value for a bit of extra validation?

**Pull request merged**